model/beer.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const uuidV1 = require('node-uuid').v1;
+const storage = require('../lib/storage.js');
+const createError = require('http-errors');
+
+
+const Beer = module.exports = function(opts) {
+  this.name = opts.name;
+  this.type = opts.type;
+  this.strength = opts.strength;
+  this.id = uuidV1();
+  this.created = new Date();
+};
+
+Beer.fetchAll = function() {
+  return storage.availIds('beers');
```

Comment:
**YOU DID NOT LOST POINTS, BUT YOU SHOULD STILL CORRECT IT ANYWAY**

Be careful with different capitalizations. You have the same error here that you did in code review with inconsistent capitalizations of ID and that was breaking your code.